lib/internal/streams/end-of-stream.js

```diff
@@ -61,9 +71,12 @@ function eos(stream, options, callback) {
   const readable = options.readable ?? isReadableNodeStream(stream);
   const writable = options.writable ?? isWritableNodeStream(stream);
 
+  if (!isNodeStream(stream) && !isReadableStream(stream) && !isWritableStream(stream)) {
+    throw new ERR_INVALID_ARG_TYPE('stream', ['ReadableStream', 'WritableStream', 'Stream'], stream);
+  }
+
   if (!isNodeStream(stream)) {
```

Comment:
If is readablestream

Reply:
didn't get you?

Reply:
Rather than assuming it's a web stream if it's not a node stream it's more clear to reverse the check. If it's a web stream then it's a web stream.

Reply:
Ok I have updated the code style could you check?